third_party/ibis/ibis_mssql/__init__.py

```diff
@@ -30,6 +33,7 @@ class Backend(BaseAlchemyBackend):
     name = "mssql"
     compiler = MsSqlCompiler
     supports_create_or_replace = False
+    LOCK_SQL = "WITH (ROWLOCK)"
```

Comment:
Does this mean we are locking tables when we select from them?

I don't understand SQL Server very well. Is it that readers lock tables and this manages it in some way?

Reply:
Bringing here the documentation reference to help us understand it better: https://learn.microsoft.com/en-us/sql/t-sql/queries/hints-transact-sql-table?view=sql-server-ver17#rowlock